frontend/public/components/utils/nav-title.tsx

```diff
@@ -36,9 +36,12 @@ export const NavTitle = connectToModel((props: NavTitleProps) => {
   const {kind, kindObj, detail, title, menuActions, buttonActions, obj, breadcrumbsFor, style} = props;
   const data = _.get(obj, 'data');
   const isCSV = !_.isEmpty(data) && referenceFor(data) === referenceForModel(ClusterServiceVersionModel);
+  const isCSC = !_.isEmpty(data) && referenceFor(data) === referenceForModel(ClusterServiceClassModel);
```

Comment:
Looks like we should have a `Set<GroupVersionKind>` that contains the resources that need special titles.